src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_actions.py

```diff
@@ -251,7 +254,54 @@ def _handle_container_ssh_file(**kwargs):
         return
 
     args = kwargs['args']
-
-    args.ssh_key_value = read_content_if_is_file(args.ssh_key_value)
+    string_or_file = args.ssh_key_value
+    content = string_or_file
+    if os.path.exists(string_or_file):
+        logger.info('Use existing SSH public key file: %s', string_or_file)
+        with open(string_or_file, 'r') as f:
+            content = f.read()
+    elif not _is_valid_ssh_rsa_public_key(content) and args.generate_ssh_keys:
+        #figure out appropriate file names:
+        #'base_name'(with private keys), and 'base_name.pub'(with public keys)
+        public_key_filepath = string_or_file
+        if public_key_filepath[-4:].lower() == '.pub':
+            private_key_filepath = public_key_filepath[:-4]
+        else:
+            private_key_filepath = public_key_filepath + '.private'
+        logger.info('Creating SSH key files: %s,%s', private_key_filepath, public_key_filepath)
```

Comment:
I am thinking about making this as a `warn` rather a `info`, due to the importance of this file.
Any objections\comments?